grassroots-backend/src/util/LastDependencyUpdateTime.ts

```diff
@@ -0,0 +1,2 @@
+// This file is updated when grassroots-shared is done building to trigger a rebuild in watch mode.
+export const LAST_DEPENDENCY_UPDATE_TIME = 209989.771846;
```

Comment:
Can this be git ignored? I suspect if it's not it will change in every PR going forward